eox_tenant/edxapp_wrapper/backends/oauth_dispatch_j_v1.py

```diff
@@ -0,0 +1,10 @@
+""" Backend abstraction. """
+
+from openedx.core.djangoapps.oauth_dispatch.dot_overrides.validators import (  # pylint: disable=import-error
+    EdxOAuth2Validator,
+)
+
+def get_edx_oauth2_validator_class():
+    """ Backend to get EdxOAuth2Validator class
```

Comment:
the double quotes must be in the same line if the text has only one line

Reply:
ty, solved